cmd/e2e_test.go

```diff
@@ -17,6 +17,7 @@ import (
 	_ "github.com/redhat-appstudio/e2e-tests/tests/integration-service"
 	_ "github.com/redhat-appstudio/e2e-tests/tests/release"
 	_ "github.com/redhat-appstudio/e2e-tests/tests/release/pipelines"
+	_ "github.com/redhat-appstudio/e2e-tests/tests/release/service"
```

Comment:
With this PR, we now have a release suite, release-pipelines suite, and release-service suite. I do not think that is good. I think there should not be a service suite and the existing release suite should be used in its place @davidmogar

Reply:
My understand is the release suite will be replaced by release-service and release-pipelines at last(after all of the release/*.go files being moved to release-service or release-pipelines).

Reply:
This. We will have only two suites and the change will be produced by Cuiping. I suggested to use `release` here as Cuiping didn't produced the change yet by that time.